sdk/appconfiguration/tests.yml

```diff
@@ -6,7 +6,7 @@ jobs:
       ServiceDirectory: appconfiguration
       EnvVars:
         AZURE_TEST_MODE: RECORD
-        AZURE_APPCONFIG_CONNECTION_STRING: $(java-azconfig-test-connection-string)
+        AZURE_APPCONFIG_CONNECTION_STRING: $(AZURE_APPCONFIG_CONNECTION_STRING)
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
```

Comment:
Doesn't the ARM template need to setup and configure everything for AAD? Right now this will attempt to use an existing application which is unlikely to have the correct access to the built AppConfiguration resource, unless these resources are constructed with a global access group assigned to them.